config/locales/en.yml

```
@@ -233,6 +233,11 @@ en:
         Do not reply to this email - it’s an automatic message from an unmonitored account.
   coronavirus_form:
     submit_and_next: "Continue"
+    api_errors:
+      400: "Bad Request | E.g. missing query parameter, malformed syntax."
+      401: "Unauthorized | The client has not provided authentication or incorrect authentication."
+      404: "Not Found | The server has not found anything matching the Request-URI."
+      500: "Internal Server Error| Generic internal server error."
```

```suggestion
      500: "Internal Server Error | Generic internal server error."
```

However, I think a content person / service designer will likely want to write these.

---

https://design-system.service.gov.uk/patterns/service-unavailable-pages/

---

Have made changes to the copy and no longer use these responses from OS.